### DISCUSSION TITLE
Notify each species detection not in an eBird Life List?

### DISCUSSION THREAD

**Discussion by sylvar** [mcguirepr89/BirdNET-Pi#807] [Ideas]
I'm not sure how easy eBird would make this for us, but it would be cool if there were a way to "Notify each lifer detection", comparing against a locally cached list of species. 

BirdNET-Pi could prompt the user to upload the results of https://ebird.org/lifelist?r=world&time=life&fmt=csv for the best accuracy, or provide a URL like https://ebird.org/profile/MTQ3NzM0/US (which as far as I know can't be made to show a world life list) for updating on a nightly schedule. 

We might want to introduce a user-definable $lifer variable that could be included in the notification, being null unless the detection seems to be a lifer.

**Comment by ehpersonal38** [comment]
Interesting idea! But for clarity, do you want a "notify each species detection **NOT** in an eBird life list?" or "notify each species detection in an eBird life list?" 

In the case of the former, I guess a quick way to do this would be to add all your lifer species to the new "Exclude these species" Apprise option, so that only non-lifers ever get notified. 

![chrome_SkbYN9dbEQ](https://user-images.githubusercontent.com/103586016/228095763-84827cd7-0f0d-4af1-b4f6-30701d013730.png)

**Comment by sylvar** [comment]
I want the option to get notified when BirdNET-Pi thinks there might be
something nearby that I haven't personally observed yet.

I also want to be able to make the notification say something like "WOW!
New lifer if it checks out: Bachman's warbler (Vermivora bachmanii) was
just detected with a confidence of 94.7% (etc.)", where I would be able to
define $lifer = "WOW! New lifer if it checks out: ".

**Reply by matthew73210** [reply]
I like the idea. Got pinged with an osprey yesterday in the snowy mountains. Never been seen with ebird but it comes under the 0.01 in the settings cus there is a huge body of water about 100 km down the valley.
